config/crds/faros_v1alpha1_clustergittrackobject.yaml

```diff
@@ -6,6 +6,13 @@ metadata:
     controller-tools.k8s.io: "1.0"
   name: clustergittrackobjects.faros.pusher.com
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.conditions[?(@.type=="ObjectInSync")].status
+    name: InSync
+    type: string
+  - JSONPath: .metadata.creationTimestamp
```

Comment:
It's funky that we need this, @icelynjennings should we add a comment to explain why this is necessary?

Reply:
@JoelSpeed will it not eventually be lost here, given it's a generated file?

Reply:
That's a very good point, Ignore me!!!